AlfrescoKit/AlfrescoKit/Cells/AKAlfrescoNodeCell.m

```diff
@@ -0,0 +1,57 @@
+/*
+ ******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile SDK.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *****************************************************************************
+ */
+
+#import "AKAlfrescoNodeCell.h"
+#import "AKUtility.h"
+
+@implementation AKAlfrescoNodeCell
+
+- (void)updateCellWithAlfrescoNode:(AlfrescoNode *)node
+{
+    self.nodeTextLabel.text = node.name;
+    
+    if ([node isKindOfClass:[AlfrescoFolder class]])
+    {
+        self.nodeImageView.image = [UIImage imageNamed:@"small_folder.tiff"];
```

Comment:
Since the library is created for the Mac target, the COMBINE_HIDPI_IMAGES flag was set. I need to set this back to NO and remove .tiff references. Good spot.